src/sdk/accountspace_fs.js

```diff
@@ -620,7 +615,7 @@ class AccountSpaceFS {
     }
 
     async _check_username_already_exists(action, username) {
-        const name_exists = await this.config_fs.is_account_exists({ name: username });
+        const name_exists = await this.config_fs.is_account_exists_by_name(username, undefined);
```

Comment:
`const name_exists = await this.config_fs.is_account_exists_by_name(username, undefined);`
`const is_user_account_exists = await this.config_fs.is_account_exists_by_name(username);`
Why sometimes do I see in this file a call with a second argument with undefined and sometimes no?